README.md

```diff
@@ -87,105 +133,21 @@ http {
 }
 ```
 
-## How to Use
-
-### Directives
-
-#### Available in `http/server/location` contexts
-
-**`otel_trace`** `on | off | “$var“;`
-
-The argument is a “complex value”, which should result in `on`/`off` or `1`/`0`. Default is `off`.
-
-**`otel_trace_context`** `ignore | extract | inject | propagate;`
-
-Defines how to propagate traceparent/tracestate headers. `extract` uses existing trace context from request. `inject` adds new context to request, rewriting existing headers if any. `propagate` updates existing context (i.e. combines `extract` and `inject`). `ignore` skips context headers processing. Default is `ignore`.
-
-**`otel_span_name`** `name;`
-
-Default is request’s location name.
-
-**`otel_span_attr`** `name “$var”;`
-
-If name starts with `http.(request|response).header.` the type of added attribute will be `string[]` to match semantic conventions (i.e. header value will be represented as a single element array). Otherwise, the attribute type will be `string`.
-
-#### Available in `http` context
-
-**`otel_exporter`**`;`
-
-Defines how to export tracing data. There can only be one `otel_exporter` directive in a given `http` context.
-
-```nginx
-otel_exporter {
-    endpoint “host:port“;
-    interval 5s;         # max interval between two exports
-    batch_size 512;      # max number of spans to be sent in one batch per worker
-    batch_count 4;       # max number of pending batches per worker, over the limit spans are dropped
-}
-```
-
-**`otel_service_name`** `name;`
-
-Sets `service.name` attribute of OTel resource. By default, it is set to `unknown_service:nginx`.
-
-### Available in `otel_exporter` context
-
-**`endpoint`** `"host:post";`
-
-Defines exporter endpoint `host` and `port`. Only one endpoint per `otel_exporter` can be specified.
-
-**`interval`** `5s;`
-
-Maximum interval between two exports. Default is `5s`.
-
-**`batch_size`** `512;`
-
-Maximum number of spans to be sent in one batch per worker. Detault is 512.
-
-**`batch_count`** `4;`
-
-Maximum number of pending batches per worker, over the limit spans are dropped. Default is 4.
-
-### Variables
-
-`$otel_trace_id` - trace id.
-
-`$otel_span_id` - current span id.
-
-`$otel_parent_id` - parent span id.
-
-`$otel_parent_sampled` - `sampled` flag of parent span, `1`/`0`.
-
-### Default span [attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md)
-
-`http.method`
-
-`http.target`
-
-`http.route`
-
-`http.scheme`
-
-`http.flavor`
-
-`http.user_agent`
-
-`http.request_content_length`
-
-`http.response_content_length`
-
-`http.status_code`
-
-`net.host.name`
+## Collecting and Viewing Traces
```

Comment:
You have removed all the directives documentation here. I would like to see it here on GH too, I would like to be able to have a quick reference of available configuration options in the Readme file. alternatively may be a complete example of all directives would be helpful. Although, that's my taste. 

PS: I think that many external folks may be open to contributing to the docs/examples via the GitHub repo, but contributing to nginx.org is a completely different story. and the directive documentation I may suspect is hand-crafted there too. the source of truth is this repo. Just my 5 cents about the complexity around directive docs.

Reply:
This is a link to the nginx.org docs file for Otel module: http://hg.nginx.org/nginx.org/file/tip/xml/en/docs/ngx_otel_module.xml

Reply:
I'd say we should try to keep one "reference doc" and right now it's at the nginx.org, though it doesn't have to be this way.

Reply:
Agreed, however, for simplicity of maintenance, nginx.org is currently the source of truth. This may change, and if it does, we can update the readme with relevant directives.